### PR TITLE
Wiki2docs

### LINKED ISSUE #635: Move readme content to (well structured) wiki
Just skimmed through the readme and it is really cluttered with a lot of information. The TOC helps but I think a lot of the stuff can be moved over to the wiki. We should keep just the important part inside the readme (like getting started and contributions etc.)

I will probably create empty wiki pages the coming days to showcase the idea that I have. Any help is appreciated in this department, as usual :)

### PR DESCRIPTION
**Pull request by vringar**
Closes #635

### CODE REVIEW

**Comment by JojiiOfficial** [comment]
Nice Work! Could you squash it into one commit?

**Comment by vringar** [comment]
This is PR is currently mostly copying stuff from either the wiki or the README. There should probably more interlinking and a better introduction.

**Comment by vringar** [comment]
> Nice Work! Could you squash it into one commit?
Sure :)

**Comment by JojiiOfficial** [comment]
No. I just think its more clean being one commit :)

**Comment by vringar** [comment]
Done!

**Review by JojiiOfficial — DISMISSED** [review]

**Comment by robinvd** [comment]
Will look at it tomorrow, a bit too big to quickly check.

**Comment by robinvd** [comment]
looks good! two things:
- did you test the CI?
- i think @SirWindfield was interested in this, so maybe wait a bit if he wants to review or merge after maybe a week if he doenst respond?

**Comment by vringar** [comment]
> looks good! two things:
> 
>     * did you test the CI?
No, because I don't know how. However I just yoinked it directly from [here](https://github.com/SirWindfield/cargo-create/blob/master/.github/workflows/docs.yml) so I'm hoping it works.
>     * i think @SirWindfield was interested in this, so maybe wait a bit if he wants to review or merge after maybe a week if he doenst respond?
Seems good to me :)

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: '0.4.2'
+
+      - name: Build book
+        run: mdbook build ./docs
+      
+      - name: Write CNAME file
+        run: "echo 'jen.zerotask.net' > ./docs/book/CNAME"
```
Comment (robinvd) [inline .github/workflows/docs.yml:27]: i think this whole step should be removed as we dont have i cname for this project (i think?)

File: docs/book.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Sven Lechner", "vringar"]
```
Comment (robinvd) [inline docs/book.toml:2]: maybe we can just make the authors something more generic like `spotifyd contributers`? what do you think @JojiiOfficial / @Spotifyd/maintainers

**Review by robinvd — CHANGES_REQUESTED** [review]

File: .github/workflows/docs.yml (hunk 1 above)
Comment (SirWindfield) [reply .github/workflows/docs.yml:27]: Yep, definitely!

File: docs/book.toml (hunk 2 above)
Comment (SirWindfield) [reply docs/book.toml:2]: Ye, I am looking into setting the contributors thing up. I already installed the bot but didn't go through the configuration yet. As for now, just set it to something like `spotifyd team & contributors` or even use the official team name `@Spotifyd/maintainers`.

File: docs/src/installation/Feature-flags.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+```bash
+cargo build --release --features "pulseaudio_backend"
+```
+
+You will need the development package for PulseAudio, as well
+as `build-essential` or the equivalent package of your distribution.
+
+### PortAudio
+
+To use PortAudio (works on macOS), compile with the `--features` flag to enable it:
```
Comment (JojiiOfficial) [inline docs/src/installation/Feature-flags.md:44]: Works on linux too. I'm using portaudio and (compiled with the right rust version) It works! Mabe we should remove the 'works on MacOS' part?

**Review by robinvd — APPROVED** [review]

**Review by JojiiOfficial — APPROVED** [review]

**Comment by robinvd** [comment]
alright, lets see if the CI works :p

**Comment by SirWindfield** [comment]
works! https://spotifyd.github.io/spotifyd/

**Comment by robinvd** [comment]
now checking the README, i dont think there currently is a link to the book

**Comment by SirWindfield** [comment]
I just added one

**Pull request merged**